packages/css/src/components/blockquote/blockquote.scss

```diff
@@ -34,6 +34,6 @@
   }
 }
 
-.ams-blockquote--inverse-color {
-  color: var(--ams-blockquote-inverse-color);
+.ams-blockquote--color-inverse {
+  color: var(--ams-blockquote-color-inverse);
```

Comment:
The name of component tokens must end with the CSS property they apply to. This should be `--ams-blockqoute-inverse-color`: block, modifier, property ([NLDS](https://nldesignsystem.nl/handboek/design-tokens/)). And likewise for other components.

Reply:
This is confusing me, I had to drop "color" from the token name, while it was there before (`"inverse-color": { "color": {}}` will become `"inverse": { "color": {}}`) but the modifier will remain to be `--inverse-color`? Whilst a button variant like "primary" is not called "variant-primary".